docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -92,6 +95,7 @@ Major versions of the .NET SDK are typically released within a few days of a Vis
 | RC 1 | 17.4 Preview 2 |
 | RC 2 | 17.4 Preview 3 |
 | 7.0.100 | 17.4.0 |
+| 8.0.100 Preview 1 | 17.6 Preview 1 |
```

Comment:
Should 17.5 be mentioned?

Reply:
There's no 8 preview that aligns with 17.5.  Not sure if there is value in mentioning that just to avoid confusion.

Reply:
Not necessary, I think it's good as-is then.